grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -632,11 +653,15 @@ class _IncidentPage extends React.Component<IncidentPageProps, IncidentPageState
     };
   };
 
-  getSilenceClickHandler = (incidentId: ApiSchemas['AlertGroup']['pk']) => {
+  getSilenceClickHandler = (incident: ApiSchemas['AlertGroup']) => {
     const { store } = this.props;
 
     return (value: number) => {
```

Comment:
can we mark this fn as `async` as it is returning Promises?

Reply:
Had to add the promise because the other caller was expecting a promise return instead of void